notebooks/work-in-progress/explore_fuel_allocation.ipynb

```diff
@@ -2,8 +2,8 @@
  "cells": [
```

Comment:
<p>What are non-associated GF &amp; BF records? Why does that mess things up here? In this particular example just based on the generator ID it seems like it might be cogeneration? At the level of discrepancy that you're seeing between the tables here, is it possible that the cogeneration units are causing the issues? Are the boiler-fuel units that have excess fuel relative to the generation fuel table involved in cogeneration? Like <strong>all</strong> the fuel is being reported in the BF table, and then it's broken out by electricity &amp; cogen in the GF table, but you're just looking at the electricity fuel consumed?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/catalyst-cooperative/pudl/pull/1096/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>